core/chain/abci.go

```diff
@@ -176,6 +205,18 @@ func (app *CoreApplication) FinalizeBlock(ctx context.Context, req *abcitypes.Fi
 		AppHash:     nextAppHash,
 	})
 
+	// increment number of proposed blocks for sla auditor
```

Comment:
Note that empty blocks don't count toward SLA if we update the report here, which is probably what we want but just flagging.